### PR TITLE
Chapter 4: Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by tuanvuP**

### CODE REVIEW

**Comment by tuanvuP** [comment]
screenshot from 2018-09-13 10-46-29

**Comment by tuanvuP** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```
[the remark this answers is not in the document]
Comment (tuanvuP) [reply app/helpers/application_helper.rb]: done

File: config/application.rb [hunk 2]
```diff
@@ -7,11 +9,18 @@
 Bundler.require(*Rails.groups)
 
 module SampleApp
+  # This cop checks for missing top-level documentation of
```
[the remark this answers is not in the document]
Comment (tuanvuP) [reply config/application.rb]: done

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
```
[the remark this answers is not in the document]
Comment (tuanvuP) [reply app/helpers/application_helper.rb]: done

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= full_title(yield(:title)) %></title>
```
[the remark this answers is not in the document]
Comment (tuanvuP) [reply app/views/layouts/application.html.erb]: done

**Comment by tuanvuP** [comment]
ready

**Comment by VuThiTranVan** [comment]
merged

**Pull request merged**